src/main/java/hudson/plugins/sonar/client/SQProjectResolver.java

```diff
@@ -77,6 +87,34 @@ public ProjectInformation get(String projectUrl, String installationName) {
     }
   }
 
+  private void getCETask(WsClient wsClient, ProjectInformation projectInfo, String ceTaskId, Float version) throws Exception {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Define and throw a dedicated exception instead of using a generic one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS00112)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make "getCETask" a "static" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS2325)